sound/soc/sof/sof-pci-dev.c

```diff
@@ -201,7 +201,7 @@ static const struct sof_dev_desc tgl_desc = {
 	.default_tplg_path = "intel/sof-tplg",
 	.default_fw_filename = "sof-tgl.ri",
 	.nocodec_tplg_filename = "sof-tgl-nocodec.tplg",
-	.ops = &sof_cnl_ops,
+	.ops = &sof_tgl_ops,
```

Comment:
don't we also need this on ICL?

Reply:
yes we do. I'm trying to verify if there's a difference in the flow compared to TGL. Will update soon

Reply:
@plbossart the flow for ICL is different for TGL. It requires using the ext manifest to know if the FW is configured for LPRO or HPRO. I need to work on the kernel-side parser for this info in the ext manifest before I can implement the flow for ICL. So for now, this change is good for TGL.